Description: enable verbose build
 The upstream Makefile does not support verbose builds yet.
Author: Michael Stapelberg <stapelberg@debian.org>
Forwarded: not-needed
Last-Update: 2013-06-18

---

Index: i3status/Makefile
===================================================================
--- i3status.orig/Makefile	2013-06-18 08:05:31.587576700 +0200
+++ i3status/Makefile	2013-06-18 08:07:55.650898415 +0200
@@ -56,18 +56,15 @@
 OBJS:=$(OBJS:.c=.o)
 
 src/%.o: src/%.c include/i3status.h
-	@$(CC) $(CPPFLAGS) $(CFLAGS) -c -o $@ $<
-	@echo " CC $<"
+	$(CC) $(CPPFLAGS) $(CFLAGS) -c -o $@ $<
 
 %.o: %.c include/%.h
-	@$(CC) $(CPPFLAGS) $(CFLAGS) -c -o $@ $<
-	@echo " CC $<"
+	$(CC) $(CPPFLAGS) $(CFLAGS) -c -o $@ $<
 
 all: i3status manpage
 
 i3status: ${OBJS}
-	@$(CC) $(LDFLAGS) -o $@ $^ $(LIBS)
-	@echo " LD $@"
+	$(CC) $(LDFLAGS) -o $@ $^ $(LIBS)
 
 clean:
 	rm -f *.o src/*.o
